public/app/features/trails/MetricScene.tsx

```diff
@@ -165,7 +165,7 @@ export class MetricActionBar extends SceneObjectBase<MetricActionBarState> {
     reportExploreMetrics('selected_metric_action_clicked', { action: 'open_in_explore' });
     this.getLinkToExplore().then((link) => {
       // We need to ensure we prefix with the appSubUrl for environments that don't host grafana at the root.
-      const url = `${config.appSubUrl}${link}`;
+      const url = `${link}`;
```

Comment:
Now we are using link directly so we can just pass it to the `window.open`. No need to define a parameter.

Reply:
Agreed! LGTM
`window.open(link, '_blank');`